packages/react-magma-docs/package.json

```diff
@@ -50,7 +50,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "gatsby build --prefix-paths",
+    "build": "GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES=true gatsby build --prefix-paths",
```

Comment:
what is this for?

Reply:
https://www.gatsbyjs.com/docs/conditional-page-builds/